.vsts-ci.yml

```diff
@@ -0,0 +1,155 @@
+trigger:
+- master
+
+variables:
+  teamName: Roslyn-Project-System
+
+phases:
+- template: /eng/build.yml
+  parameters:
+    agentOs: Windows_NT
+    queue:
+      name: Hosted VS2017
+      parallel: 2
```

Comment:
you have 4 legs of the matrix, do you want to change this to 4 or even to 99 so you don't have to fiddle with it and these always run in parallel if there are machines available?